### ISSUE TITLE
statin-simvastatin

### ISSUE THREAD

**Issue by slaulederkind** [geneontology/go-ontology#12008]
Consider making "response to statin" (GO:0036273) an additional parent to "response to simvastatin" (GO:1903491).  Thanks.

**Comment by paolaroncaglia** [comment]
Hi Stan,
Link added. 
Thanks,
Paola

Note for self: 
Leaving ticket open as we (Melanie) contacted ChEBI (Janna + Gareth) about this:

"In short, GO ha response to statin and response to simvastatin, where simvastatin should be a statin - this would allow inference that response to simvastatin is a type of response to statin. Statins are defined as a class of drug with a specific inhibitory activity (EC 1.1.1.34/EC 1.1.1.88 (hydroxymethylglutaryl-CoA reductase) inhibitor)

We checked ChEBI whether there was a statin term, parent of simvastatin, but couldn't find anything. However, ChEBI has the role 'EC 1.1.1.34/EC 1.1.1.88 (hydroxymethylglutaryl-CoA reductase) inhibitor': http://www.ebi.ac.uk/chebi/chebiOntology.do?chebiId=CHEBI:35664, which itself has synonym 'statin'

The issue is that the role and the compound bearing the role seem to be conflated: definition of CHEBI:35664 reads 'A compound that inhibits HMG-CoA reductases [...]' and the class has members based on those molecule bearing the role, such as fluvastatin, http://www.ebi.ac.uk/chebi/searchId.do?chebiId=5137.

Ideally, I think we would want to have a 'statin' class defined as 'molecular entity and has_role some CHEBI:35664. GO could then directly reuse this term and you would have the inference to the subclasses happening automatically.

Does that make sense?"

Janna's reply:

"Gareth is not in the office at the moment, I think this is something we need to discuss when he is back. These structure/role combinations are always tricky but we will find a solution,"

**Comment by paolaroncaglia** [comment]
Gareth wrote:

“We how have a new entity: statin (CHEBI:87631), which is classed as
    is a organooxygen compound (CHEBI:36963);
    has role anticholesteremic drug (CHEBI:35821);
    has role EC 1.1.1.34/EC 1.1.1.88 (hydroxymethylglutaryl-CoA reductase) inhibitor (CHEBI:35664)

The new term has been sub-divided into:
    statin (naturally occurring) (CHEBI:87632);
    statin (semi-synthetic) (CHEBI:87633); and
    statin (synthetic) (CHEBI:87635)
each of which has been populated with the appropriate drugs.

To avoid confusion, I have deleted the terms 'statin' and 'statins' as synonyms for EC 1.1.1.34/EC 1.1.1.88 (hydroxymethylglutaryl-CoA reductase) inhibitor (CHEBI:35664). Also for CHEBI:35664, I have deleted the relationship is a anticholesteremic drug (CHEBI:35821), since this is now covered by  statin (CHEBI:87631).”

Currently, GO:0036273 ‘response to statin’ is logically defined as

[Term]
id: GO:0036273
name: response to statin
namespace: biological_process
def: "Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of a statin stimulus, a compound that inhibits HMG-CoA reductases." [CHEBI:35664, GOC:hp]
synonym: "response to HMG-CoA reductase inhibitor" EXACT [CHEBI:35664]
synonym: "response to hydroxymethylglutaryl-CoA reductase inhibitor" EXACT [CHEBI:35664]
is_a: GO:0042493 {is_inferred="true"} ! response to drug
intersection_of: GO:0050896 ! response to stimulus
intersection_of: has_input GOCHE:35664 ! substance with hydroxymethylglutaryl-CoA reductase inhibitor application 

I will change the gloss of the text def to read
“…a statin stimulus. Statins are organooxygen compounds whose structure is related to compactin (mevastatin) and which may be used as an anticholesteremic drug due its EC 1.1.1.34/EC 1.1.1.88 (hydroxymethylglutaryl-CoA reductase) inhibitory properties.”
and remove the dbxref CHEBI:35664

I will change the logical definition to  
intersection_of: GO:0050896 ! response to stimulus
intersection_of: has_input CHEBI:87631 ! statin 

and I will change the synonyms’ scope from EXACT to RELATED.
